test/xspec-variable_stylesheet.xspec

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description
+	xmlns:my="http://example.org/ns/my"
+	xmlns:myv="http://example.org/ns/my/variable"
+	xmlns:t="http://www.jenitennison.com/xslt/xspec"
+	xmlns:x="http://www.jenitennison.com/xslt/xspec"
+	xmlns:xs="http://www.w3.org/2001/XMLSchema"
+	stylesheet="xspec-variable-tested.xsl">
+	<!-- Variables used only in this test file are in a dedicated namespace to prevent conflict with
+	 global parameters or global variables in the file to be tested. -->
+
+	<!--XQuery needs mode=test:generate-variable-declarations fix.
+	 TODO: When that's available, move "Combination of @select and element content" scenario to the
+	 "Variables can be defined in different ways." scenario in xspec-variable.xspec. -->
```

Comment:
Looks like this TODO is now doable. (perhaps thanks to #427)

Reply:
Thanks for letting me know. I uploaded https://github.com/xspec/xspec/pull/475/commits/d9840e07d10e5158154a2a76a34a7d6715a1da4c with that change.